data/hai outfits.txt

```diff
@@ -608,4 +608,4 @@ outfit "Pulse Rifle"
 	"capture attack" .8
 	"capture defense" 1.3
 	"unplunderable" 1
-	description "The Pulse Rifle was first developed by the Hai to help protect their ships from boarders during the Korath raids. While it has since been adapted to not be lethal outright, if several bolts hit a target rapidly, it will shut down the nervous system, killing the target."
+	description "The Pulse Rifle was first developed by the Hai to help protect their ships from boarders during the Korath raids. While it has since been adapted to more effectively incapacitate its target, if several bolts land in quick succession, the accumulated charge has a lethal effect on most beings' nervous systems."
```

Comment:
`beings` doesn't sound right. Do what the biodefenses did and use `species` instead.

Reply:
that does sound better